runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -1959,6 +1982,24 @@ TR_J9InlinerPolicy::inlineUnsafeCall(TR::ResolvedMethodSymbol *calleeSymbol, TR:
       case TR::sun_misc_Unsafe_putObjectOrdered_jlObjectJjlObject_V:
          return createUnsafePutWithOffset(calleeSymbol, callerSymbol, callNodeTreeTop, callNode, TR::Address, false, true, true);
 
+      // FIXME: Update createUnsafePutWithOffset signature to have isVolatile, isOrdered, isUnaligned as enum
+      case TR::jdk_internal_misc_Unsafe_getCharUnaligned:
+         return createUnsafeGetWithOffset(calleeSymbol, callerSymbol, callNodeTreeTop, callNode, TR::Int8, /*needsNullCheck*/false, /*isUnaligned*/true);
+      case TR::jdk_internal_misc_Unsafe_getShortUnaligned:
+         return createUnsafeGetWithOffset(calleeSymbol, callerSymbol, callNodeTreeTop, callNode, TR::Int16, /*needsNullCheck*/false, /*isUnaligned*/true);
+      case TR::jdk_internal_misc_Unsafe_getIntUnaligned:
+         return createUnsafeGetWithOffset(calleeSymbol, callerSymbol, callNodeTreeTop, callNode, TR::Int32, /*needsNullCheck*/false, /*isUnaligned*/true);
+      case TR::jdk_internal_misc_Unsafe_getLongUnaligned:
+         return createUnsafeGetWithOffset(calleeSymbol, callerSymbol, callNodeTreeTop, callNode, TR::Int64, /*needsNullCheck*/false, /*isUnaligned*/true);
```

Comment:
Contrary to the comments, these calls are passing `isVolatile=false`, `needNullCheck=true`, and the default `isUnaligned=false`. It looks like `isVolatile=false` is missing between the datatype and `needNullCheck`